### PR TITLE
[hadoop][node_manager] Add node_manager data stream for Hadoop.

### PR DESCRIPTION
**Pull request by yug-elastic**
## What does this PR do?

- Generated the skeleton of Hadoop integration package.
- Added 1 data stream ( Node Manager Metrics )
- Added data collection logic.
- Added the ingest pipelines.
- Mapped fields according to the ECS schema and added Fields metadata in the appropriate yml files.
- Added system test cases.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^8.0.0`).

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/hadoop directory.
- Run the following command to run tests. 
> elastic-package test 

Note: We have added all the visualizations for Hadoop in the same PR as splitting them would be tedious and create more conflicts for the other data stream PRs.

### CODE REVIEW

File: packages/hadoop/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+title: "Hadoop"
+version: 0.1.0
+license: basic
+description: "This Elastic integration collects metrics from hadoop."
+type: integration
+categories:
+  - containers
+  - datastore
+conditions:
+  kibana.version: "^8.0.0"
```
Comment (yug-elastic) [inline packages/hadoop/manifest.yml:12]: This will be updated with 8.2.0 after testing this integration on 8.2.0.

**Comment by yug-elastic** [comment]
This PR is a split of https://github.com/elastic/integrations/pull/2614 as discussed over the comment https://github.com/elastic/integrations/pull/2614#issuecomment-1082704609

**Review by mtojek — COMMENTED** [review]
This PR contains all data streams instead of "node_manager". Can we strip it to contain only one?

**Comment by yug-elastic** [comment]
> This PR contains all data streams instead of "node_manager". Can we strip it to contain only one?

Updated the PR, please take a look at your convenience.

**Review by mtojek — COMMENTED** [review]
Could we also extract Kibana objects to a separate follow-up PR? This one will shrink more.

**Comment by yug-elastic** [comment]
> Could we also extract Kibana objects to a separate follow-up PR? This one will shrink more.

Sure, do you want us to remove all the visualizations in another PR or just keep the visualizations for the NameNode here and remove other visualizations?

**Comment by mtojek** [comment]
I think that I suggest keeping them "parked" in a separate PR so that we can focus on particular data streams, and then iterate with visualizations. I hope that it isn't a big deal to extract them.

**Comment by yug-elastic** [comment]
> I think that I suggest keeping them "parked" in a separate PR so that we can focus on particular data streams, and then iterate with visualizations. I hope that it isn't a big deal to extract them.

Definitely, it's not a big deal. Let's do that.

**Review by mtojek — APPROVED** [review]

File: packages/hadoop/manifest.yml (hunk 1 above)
Comment (yug-elastic) [reply packages/hadoop/manifest.yml]: Kibana version has been updated to 8.1.0 as discussed over weekly sync for all the integrations.

**Review by lalit-satapathy — APPROVED** [review]

**Pull request merged**